core/src/main/kotlin/io/fritz2/dom/tag.kt

```diff
@@ -29,4 +30,11 @@ abstract class Tag<out T : Element>(tagName: String, override val domNode: T = w
     fun <X : Element> Flow<Tag<X>>.bind(): SingleMountPoint<WithDomNode<Element>> = DomMountPoint(this, domNode)
 
     fun <X : Element> Flow<Patch<Tag<X>>>.bind(): MultiMountPoint<WithDomNode<Element>> = DomMultiMountPoint(this, domNode)
+
+    operator fun <T> T.not() = Const(this)
```

Comment:
I put it in here cause then no extra import is needed. So it's a bit easier to use ;)